gateway/index.ts

```diff
@@ -21,14 +24,15 @@ async function initServer() {
 
   await server.start();
 
-  app.use(cors({ origin: "http://localhost:7000", credentials: true }));
+  app.use(cors({ origin: originLink, credentials: true }));
```

Comment:
better. no hard coding.

Reply:
exactly. Ill explain how to leverage the env_vars to make this change.